lib/template.html

```diff
@@ -104,6 +104,10 @@
   <meta property="og:title" content="{{ seo_page_title }}" />
 {% endif %}
 
+{% if seo_author %}
+  <meta name="author" content="{{ seo_author }}" />
+{% endif %}
+
```

Comment:
Well it's ignored by Google but not by Facebook. The purpose of this change is to show up the author name of an article when you share it on Facebook. If the `meta name="author"` is not set the author name will not be displayed when you share a jekyll blog article on Facebook (Issue #96). If the author of the article has a Facebook account you can use the `meta property="article:author"` which his ID / profile URL instead but it's not always the case.  

If someone have any other suggestion to resolve this problem it would be greatly appreciated :smiley: